src/app/icd/client/DefaultICDClientStorage.cpp

```diff
@@ -295,7 +295,12 @@ CHIP_ERROR DefaultICDClientStorage::SetKey(ICDClientInfo & clientInfo, const Byt
     memcpy(keyMaterial, keyData.data(), sizeof(Crypto::Symmetric128BitsKeyByteArray));
 
     ReturnErrorOnFailure(mpKeyStore->CreateKey(keyMaterial, clientInfo.aes_key_handle));
```

Comment:
```suggestion
    // TODO : Update key lifetime once creaKey method supports it.
    ReturnErrorOnFailure(mpKeyStore->CreateKey(keyMaterial, clientInfo.aes_key_handle));
```

Does not require any actions in this PRs.